tests/unit/blueprints/datamanager/controllers/test_transform.py

```
@@ -283,3 +287,51 @@ def test_show_dedup_tab_uses_typology_but_keeps_conservation_area_spatial_flow()
+            with patch(f"{TRANSFORM_MODULE}.get_org_entity", return_value=400):
+                with patch(
+                    f"{TRANSFORM_MODULE}.get_entity_count_for_organisation_and_dataset",
+                    return_value=2,
+                ):
+                    with patch(
+                        f"{TRANSFORM_MODULE}.get_entities_for_organisation_and_dataset",
+                        return_value=[{"entity": 1}, {"entity": 2}],
+                    ):
+                        (
+                            entities,
+                            too_large,
```

Ruff (0.16.1)

[warning] Unpacked variable `too_large` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)